Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -97,8 +97,9 @@ HRESULT UpdateCollection(const ComPtr<IDWriteFactory>& dwriteFactory, const ComP
 }
 
 // Private helper, wraps around GetUserDefaultLocaleName() and returns a default of "en-us" if it fails
-static inline const wchar_t* __GetUserDefaultLocaleName(wchar_t* buf, size_t bufferSize) {
-    int defaultLocaleSuccess = GetUserDefaultLocaleName(buf, bufferSize);
+template <size_t size>
```

Comment:
This is only being used with buffers of size LOCALE_NAME_MAX_LENGTH, so there will only be one instance, but I didn't think it made sense to do any calculation or pass in a parameter when the size is known at compile-time.  Would _countof work w/ function arg?